website/content/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -1000,6 +1010,10 @@ and consider if they're appropriate for your deployment.
 
   - `targetPort` (`int: 8200`) - Sets the target port value of the service.
 
+  - `serviceIPFamilyPolicy` (`string: ""`) - The IP family and IP families options are to set the behaviour in a dual-stack environment. Omitting these values will let the service fall back to whatever the CNI dictates the defaults should be. These are only supported for kubernetes versions >=1.23. The service's supported IP family policy, can be either: SingleStack: Single-stack service. The control plane allocates a cluster IP for the Service, using the first configured service cluster IP range. PreferDualStack: Allocates IPv4 and IPv6 cluster IPs for the Service. RequireDualStack: Allocates Service .spec.ClusterIPs from both IPv4 and IPv6 address ranges.
```

Comment:
Similar thought here about just linking to the k8s docs that explains these options.